### PR TITLE
Update SetHighFallTask.md

### PR DESCRIPTION
**Pull request by LedAndris**
I tested a lot of params for p1 and p2 but wasn't able to find out any serious consistency between the flags so this just gives a hand to developers trying with ragdolls, but the second parameter is for sure the duration I'm certain about that.

### CODE REVIEW

File: TASK/SetHighFallTask.md [hunk 1]
```diff
@@ -5,16 +5,26 @@ ns: TASK
 
 ```c
 // 0x8C825BDC7741D37C 0xBBB26172
-void SET_HIGH_FALL_TASK(Ped ped, Any p1, Any p2, Any p3);
+void SET_HIGH_FALL_TASK(Ped ped, int p1, Any p2, Any p3);
 ```
 
 ```
```
Comment (technetium-cfx) [inline TASK/SetHighFallTask.md:11]: Please remove the codefence, this is only used by NativeDB imports.

File: TASK/SetHighFallTask.md [hunk 2]
```diff
@@ -5,16 +5,26 @@ ns: TASK
...
-Makes the ped ragdoll like when falling from a great height  
+Makes the ped ragdoll like when falling from a great heigh.
```
Comment (technetium-cfx) [inline TASK/SetHighFallTask.md:12]: `heigh`? 😛

File: TASK/SetHighFallTask.md [hunk 3]
```diff
@@ -5,16 +5,26 @@ ns: TASK
...
+On p1: 2, p2: 2 the ped just collapses like a bag of potato.
+Good combinations tried:
+p1: 100 p2: 1000
+p1: 100 p2: 1
+Even numbers work best
 ```
 
 ## Parameters
 * **ped**: 
+* **duration**:
```
Comment (technetium-cfx) [inline TASK/SetHighFallTask.md:27]: Parameters here must match parameter names in the function signature.

File: TASK/SetHighFallTask.md [hunk 4]
```diff
@@ -5,16 +5,26 @@ ns: TASK
...
-void SET_HIGH_FALL_TASK(Ped ped, Any p1, Any p2, Any p3);
+void SET_HIGH_FALL_TASK(Ped ped, int p1, Any p2, Any p3);
 ```
 
 ```
-Makes the ped ragdoll like when falling from a great height  
+Makes the ped ragdoll like when falling from a great heigh.
+Playing with p1 and p2 changes the animation's style and behavior.
+In most cases the ped moves backwards then fall on the ground or collapse with a very small change to it's position.
+If p1 or p2 is 1 the ped will move backwards
```
Comment (technetium-cfx) [inline TASK/SetHighFallTask.md:15]: You mention p1 is duration, but this does not suggest that. Could you elaborate on this?

File: TASK/SetHighFallTask.md [hunk 5]
```diff
@@ -5,16 +5,26 @@ ns: TASK
...
+p1: 100 p2: 1
+Even numbers work best
 ```
 
 ## Parameters
 * **ped**: 
+* **duration**: 
 * **p1**: 
 * **p2**: 
-* **p3**:
```
Comment (technetium-cfx) [inline TASK/SetHighFallTask.md:19]: p3 should still be the last param. Index is not to be changed.

**Review by technetium-cfx — CHANGES_REQUESTED** [review]
Left some feedback. Useful information to have, but should be a bit clearer 🙂

File: TASK/SetHighFallTask.md (hunk 5 above)
Comment (LedAndris) [reply TASK/SetHighFallTask.md:19]: Oh I see, thank you (that's why I used p1 and p2)

**Comment by LedAndris** [comment]
Will keep these informations in mind, thank you for your feedback.

**Comment by technetium-cfx** [comment]
You requested a review, but no changes have been made yet 😛

**Comment by LedAndris** [comment]
Thats strange, I'm quite sure I made the requested changes, but this is probably ma fault so let me retry

**Comment by technetium-cfx** [comment]
Still nothing!

**Comment by LedAndris** [comment]
Left a commit, I suck with this part of Github, sorry about this

**Pull request merged**